<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Scanner</class>
 <widget class="QWidget" name="Scanner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scanner</string>
  </property>
  <widget class="QLineEdit" name="hostnameEntry">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>localhost</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="portEntry">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>22, 80, 443</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>99</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Scan</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="resultWindow">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>hostname:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>ports:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>results:</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
